docs/internal/docs.rst

```diff
@@ -88,7 +88,7 @@ RST/Sphinx Usage
 * `One sentence per line should be perfered. <https://rhodesmill.org/brandon/2012/one-sentence-per-line/>`__
   This makes it easier to see what changed in a ``git diff`` or GitHub PR and easier to move sentences around in editors like Vim.
   It also avoids inconsistencies involving what the maximum line length is.
-  This might make it more annoying to read the documentation raw, but that's not the indented way to do so anyway.
+  This might make it more annoying to read the documentation raw, but that's not the intended way to do so anyway.
```

Comment:
Just as one more data point -- I think I've only ever read it as a plaintext file, not rendered.

Reply:
Yes, and that's fine. The intention in "intended" is what's going to be most friendly for end-users.